spec/examples/metal/keyspace_spec.rb

```diff
@@ -104,6 +104,27 @@
     end
   end
 
+  describe "#client_compression" do
+    let(:client_compression) { :lz4 }
+    let(:client_mock) do
+      client_mock = mock
+      client_mock.expects(:connect).returns(true)
+      client_mock
+    end
+    let(:connect) do
+      Cequel.connect host: Cequel::SpecSupport::Helpers.host,
+          port: Cequel::SpecSupport::Helpers.port,
+          client_compression: client_compression
+    end
+    it "client compression settings get extracted correctly for sending to cluster" do
+      expect(connect.client_compression).to eq client_compression
+    end
+    it 'properly passes compression seetting to the driver' do
+      Cassandra.expects(:cluster).with(has_entries(compression: :lz4)).returns(client_mock)
+      expect(connect.client).to eq true
```

Comment:
Is there anyway to verify this without a mock? Does the client object expose a compression attr, for example? I'd rather not use mocking if there is any other option since they basically just test that the implementation hasn't been changed.